### PR TITLE
Crpl 2865 put endpoint lesson

### PR DESCRIPTION
**Pull request by danstocker**

### CODE REVIEW

File: io/http/server/utils/json request body parser/d61c432a-5b9e-483c-a103-5bb0c8f69748.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
+{
+  "name": "io/http/server/utils/JSON request body parser",
+  "description": "Parses the body of an HTTP request as JSON.",
+  "interface": {
+    "inputs": {
+      "f522d573-49d0-45ab-ac2a-e8d4e576d60b": {
+        "type": "{\"body\": \"string\"}",
+        "name": "request",
+        "description": ""
```
Comment (csabee1979) [inline io/http/server/utils/json request body parser/d61c432a-5b9e-483c-a103-5bb0c8f69748.json:9]: Should has description.

File: io/http/server/utils/json request body parser/d61c432a-5b9e-483c-a103-5bb0c8f69748.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      }
+    },
+    "outputs": {
+      "c76bd5f3-cbf0-414e-85f2-125c06e2ca12": {
+        "type": "any",
```
Comment (csabee1979) [inline io/http/server/utils/json request body parser/d61c432a-5b9e-483c-a103-5bb0c8f69748.json:14]: Should has description.

File: tutorials/get request/get sum handler/45e20f72-9709-4b81-897f-c21ee6be09fa.json [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+              "cd2cfdb3-e99f-4836-b743-42a5834316f7": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "iconId": "11e997ab-33d0-473a-aa56-1e2fe4d30d5c",
+  "attributes": {
+    "internal": true
```
Comment (csabee1979) [inline tutorials/get request/get sum handler/45e20f72-9709-4b81-897f-c21ee6be09fa.json:120]: Should be locked.

**Review by csabee1979 — APPROVED** [review]

**Review by lvass74 — CHANGES_REQUESTED** [review]

File: data/shared store/98a9e940-ff79-4331-b30e-ce1970390ad0.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "es6": {
+      "inputFunctions": {
+        "df286b60-2beb-4138-a6b2-667612bccc17": "const {storeId, data: value} = data;\nif (value === undefined) {\n  // reading\n  const store = shared[storeId];\n  if (store === undefined) {\n    outputs.error({\"error\": `Store ${storeId} not found.`}, tag);\n  } else {\n    outputs.data(store, tag);\n  }\n} else {\n  // writing\n  shared[storeId] = value;\n  outputs.written(null, tag);\n}"
+      }
+    }
+  },
+  "interface": {
+    "inputs": {
+      "df286b60-2beb-4138-a6b2-667612bccc17": {
+        "type": "{\"storeId\": string, \"data\": any}",
```
[the remark this answers is not in the document]
Comment (danstocker) [reply data/shared store/98a9e940-ff79-4331-b30e-ce1970390ad0.json]: fixed

File: data/shared store/98a9e940-ff79-4331-b30e-ce1970390ad0.json [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+  "implementations": {
+    "es6": {
+      "inputFunctions": {
+        "df286b60-2beb-4138-a6b2-667612bccc17": "const {storeId, data: value} = data;\nif (value === undefined) {\n  // reading\n  const store = shared[storeId];\n  if (store === undefined) {\n    outputs.error({\"error\": `Store ${storeId} not found.`}, tag);\n  } else {\n    outputs.data(store, tag);\n  }\n} else {\n  // writing\n  shared[storeId] = value;\n  outputs.written(null, tag);\n}"
```
[the remark this answers is not in the document]
Comment (danstocker) [reply data/shared store/98a9e940-ff79-4331-b30e-ce1970390ad0.json]: fixed

**Review by lvass74 — APPROVED** [review]

**Pull request merged**